Reservation cancellation with charge percentage on cancellation condi…

rogierpijpers opened the pull request:
…tions

rogierpijpers committed:
Reservation cancellation with charge percentage on cancellation conditions

jeroenknoester reviewed (approved):
onder voorwaarde unittest

rogierpijpers committed:
Added unit testing for domain class

rogierpijpers committed:
Controller tests to check that receptionist can set free of charge but guest can not.

Thomvdm commented on src/test/java/com/capgemini/domain/ReservationCancellationTest.java:
```diff
@@ -0,0 +1,68 @@
+    }
+
+    @Test
+    public void testCancellationC0Percent(){
```
Staat die C0 voor iets of moet dat gewoon 0 zijn?

Thomvdm commented on src/test/java/com/capgemini/domain/ReservationCancellationTest.java:
```diff
@@ -0,0 +1,68 @@
+    }
+
+    @Test
+    public void testCancellationhargeFree(){
```
Typo

rogierpijpers committed:
Typos
